src/System.Globalization/tests/Invariant/InvariantMode.cs

```diff
@@ -784,5 +785,14 @@ public void GetUnicode(string ascii, int index, int count, string expected)
             }
             Assert.Equal(expected, new IdnMapping().GetUnicode(ascii, index, count));
         }
+
+        [Fact]
+        public void TestHashing()
```

Comment:
Better name? Maybe like `TestTurkishITurkishHashMatchesOrdinalHash`

Reply:
The intention here is not testing the Turkish but the intention is, in the invariant mode we'll not do cultural hashing and instead we'll do it ordinal.